bin/player.rb

```diff
@@ -0,0 +1,9 @@
+	class Player
+        attr_reader :name, :symbol
+
+        def initialize
+            @name = gets.chomp()
+            @symbol = gets.chomp()
```

Comment:
:bulb: Let's avoid the use of `gets` in this class. The **`main.rb` file is the only place where `puts` and `get.chomp` can be used** :blush: 

### Remember
> Add **executable file** named `main.rb` to bin directory. This file should be responsible for user interaction. This is the only one place where `puts` and `get.chomp` can be used.